### PR TITLE
[RHPAM-4481] - Update EAP version to 7.4.6 on RHPAM Images

### PR DESCRIPTION
**Pull request by spolti**
Signed-off-by: spolti <fspolti@redhat.com>
See: https://issues.redhat.com/browse/RHPAM-4481
Cherry picks: https://github.com/jboss-container-images/rhpam-7-openshift-image/pull/638

Thanks for submitting your Pull Request!

Please make sure your PR meets the following requirements:

- [x] Pull Request title is properly formatted: `[RHPAM-XYZ] Subject`
- [x] Pull Request contains link to the JIRA issue
- [x] Pull Request contains description of the issue
- [x] Pull Request does not include fixes for issues other than the main ticket
- [x] Attached commits represent units of work and are properly formatted
- [x] You have read and agreed to the Developer Certificate of Origin (DCO) (see `CONTRIBUTING.md`)
- [x] Every commit contains `Signed-off-by: Your Name <yourname@example.com>` - use `git commit -s`

### CODE REVIEW

File: businesscentral-monitoring/branch-overrides.yaml [hunk 1]
```diff
@@ -14,11 +14,11 @@ modules:
     - name: jboss-eap-modules
       git:
         url: https://github.com/jboss-container-images/jboss-eap-modules.git
-        ref: EAP_744_CR2
+        ref: EAP_746_CR2_2
     - name: jboss-eap-7-image
       git:
         url: https://github.com/jboss-container-images/jboss-eap-7-image.git
-        ref: EAP_744_CR2
+        ref: EAP_LP_746_CR1
```
Comment (sutaakar) [inline businesscentral-monitoring/branch-overrides.yaml:21]: What does LP mean?
Does it mean that it isn't released yet?

**Review by sutaakar — APPROVED** [review]

File: businesscentral-monitoring/branch-overrides.yaml (hunk 1 above)
Comment (spolti) [reply businesscentral-monitoring/branch-overrides.yaml:21]: IIRC it stands for the tags created from the legacy branch, @luck3y can confirm it.

Comment (luck3y) [reply businesscentral-monitoring/branch-overrides.yaml:21]: LP == "layered product", we just use those for tags created in between our releases when you ask for them :)

Comment (spolti) [reply businesscentral-monitoring/branch-overrides.yaml:21]: Thanks @luck3y.

**Pull request merged**